plugin.rb

```diff
@@ -18,6 +18,7 @@
 
 register_asset "stylesheets/common/resolution-form.scss"
```

Comment:
I think now we can use
`[
  "stylesheets/common/resolution-form.scss",
  "stylesheets/common/landing.scss",
  "stylesheets/common/communities-page.scss"
].each { |path| register_asset path }`

Reply:
see https://github.com/fs/communitarian-discourse-plugin/pull/19 @IlgizGilmanov